ui-performance/index.md

```diff
@@ -121,27 +121,51 @@ Gallery example:
 <center>The graphs produced by the performance overlay.<br>The vertical green
 bars represent the current frame.</center><br>
 
-Each graph represents the last 300 frames for the following two threads used
-in every Flutter app:
+Flutter uses several threads to do its work. You have direct control
+over what happens in the UI thread, but no direct access to the other
```

Comment:
"All you Dart code runs on the UI thread. And although you have no direct access to any other thread, your actions on the UI thread have performance consequences on other threads.". Or something like that. "direct control" seemed odd since Dart code doesnt know about threads.

Reply:
Done